cypress/support/elements/common/PrimaryWorkspace.js

```diff
@@ -84,8 +96,9 @@ class RightNav{
       return cy.get('.list.'+section+' [data-test='+section+'-list-items] .footer');
     }
 
-    starCanvasItem(tab,section,title){
-        cy.getCanvasItemTitle(section).contains(title).parent().parent().siblings('.icon-holder').find('.icon-star').click();
+    starCanvasItem(tab, section,title){
```

Comment:
This should also probably be in the resource panel elements.

Reply:
Done.